sandbox-backend/src/main/java/com/exadel/sandbox/team5/entity/Discount.java

```diff
@@ -0,0 +1,57 @@
+package com.exadel.sandbox.team5.entity;
+
+
+import lombok.*;
+
+import javax.persistence.*;
+import java.io.Serializable;
+import java.util.Date;
+import java.util.HashSet;
+import java.util.Set;
+
+@Setter
+@Getter
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+
+@Entity
+@Table(name = "discount")
+public class Discount extends AuditableEntity {
+
+    @ManyToMany(cascade = { CascadeType.ALL })
+    @JoinTable(
+            name = "discount_tag",
+            joinColumns = { @JoinColumn(name = "discountId") },
+            inverseJoinColumns = { @JoinColumn(name = "tagId") }
+    )
+    Set<Tag> tags = new HashSet<>();
+
+    @ManyToMany(mappedBy = "discounts")
+    private Set<Employee> employee = new HashSet<>();
```

Comment:
Probably should be list. One employee should be able to use same discount multiple times

Reply:
also we should use `List<EmployeeDiscount>` instead of `List<Discount>`. It will mean that we have some history or using  discount for each selected Discount object